<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生信息管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/aufc.ico</normaloff>:/aufc.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:aliceblue;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:aliceblue;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <item row="2" column="0">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">background-color:aliceblue;</string>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="9">
            <widget class="QPushButton" name="btn_del">
             <property name="styleSheet">
              <string notr="true">QPushButton{
border-radius:6;
color:aliceblue;
background-color: rgb(142, 221, 255);
min-width: 80;
min-height: 28;
}
QPushButton:pressed{
border-radius:6;
color:aliceblue;
background-color:rgb(255, 255, 255);
min-width: 80;
min-height: 28;
}</string>
             </property>
             <property name="text">
              <string>删除</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_clear">
             <property name="styleSheet">
              <string notr="true">QPushButton{
border-radius:6;
color:aliceblue;
background-color: rgb(142, 221, 255);
min-width: 80;
min-height: 28;
}
QPushButton:pressed{
border-radius:6;
color:aliceblue;
background-color:rgb(255, 255, 255);
min-width: 80;
min-height: 28;
}
</string>
             </property>
             <property name="text">
              <string>清空学生表</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btn_moni">
             <property name="styleSheet">
              <string notr="true">QPushButton{
border-radius:6;
color:aliceblue;
background-color: rgb(142, 221, 255);
min-width: 80;
min-height: 28;
}
QPushButton:pressed{
border-radius:6;
color:aliceblue;
background-color:rgb(255, 255, 255);
min-width: 80;
min-height: 28;
}
</string>
             </property>
             <property name="text">
              <string>模拟数据</string>
             </property>
            </widget>
           </item>
           <item row="5" column="12">
            <widget class="QLabel" name="lable_cnt">
             <property name="styleSheet">
              <string notr="true">color: rgb(81, 163, 244);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="8">
            <widget class="QPushButton" name="btn_add">
             <property name="styleSheet">
              <string notr="true">QPushButton{
border-radius:6;
color:aliceblue;
background-color: rgb(142, 221, 255);
min-width: 80;
min-height: 28;
}
QPushButton:pressed{
border-radius:6;
color:aliceblue;
background-color:rgb(255, 255, 255);
min-width: 80;
min-height: 28;
}
</string>
             </property>
             <property name="text">
              <string>增加</string>
             </property>
            </widget>
           </item>
           <item row="0" column="11">
            <widget class="QLineEdit" name="lE_search">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>27</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>27</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 2px solid  rgb(142, 221, 255);
background-color: rgb(255, 255, 255);
min-width: 120;
min-height: 25;
border-radius: 5;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="12">
            <widget class="QPushButton" name="btn_find">
             <property name="styleSheet">
              <string notr="true">QPushButton{
border-radius:6;
color:aliceblue;
background-color: rgb(142, 221, 255);
min-width: 80;
min-height: 28;
}
QPushButton:pressed{
border-radius:6;
color:aliceblue;
background-color:rgb(255, 255, 255);
min-width: 80;
min-height: 28;
}</string>
             </property>
             <property name="text">
              <string>搜索</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="15">
            <widget class="QTableWidget" name="tableWidget">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(235, 251, 255);
border:none;
</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <column>
              <property name="text">
               <string>序号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>姓名</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>年龄</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>年级</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>班级</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>学号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>电话号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>微信帐号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>QQ帐号</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="0" column="10">
            <widget class="QPushButton" name="btn_update">
             <property name="styleSheet">
              <string notr="true">QPushButton{
border-radius:6;
color:aliceblue;
background-color: rgb(142, 221, 255);
min-width: 80;
min-height: 28;
}
QPushButton:pressed{
border-radius:6;
color:aliceblue;
background-color:rgb(255, 255, 255);
min-width: 80;
min-height: 28;
}</string>
             </property>
             <property name="text">
              <string>修改</string>
             </property>
            </widget>
           </item>
           <item row="0" column="6">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color:aliceblue;</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>363</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_title">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>22</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(7, 7, 7);
font: 22pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>学生信息管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_tp">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">image: url(:/qw.ico);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_yh">
            <property name="styleSheet">
             <string notr="true">color: rgb(8,8,8);
font: 10pt &quot;楷体&quot;</string>
            </property>
            <property name="text">
             <string>用户名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_sy">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;楷体&quot;;
color: rgb(81, 163, 244);</string>
            </property>
            <property name="text">
             <string>圣颜</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
color: rgb(255, 255, 255);
background-color: rgb(250, 227, 217);
border-radius:6;            
color:aliceblue;                    
background-color: rgb(81, 163, 244);           
min-width: 80;
min-height: 28;
}
QPushButton:pressed{
color: rgb(255, 255, 255);
background-color: rgb(250, 227, 217);
border-radius:6;            
color:aliceblue;                    
background-color:rgb(255, 255, 255);           
min-width: 80;
min-height: 28;
}

</string>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1088</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
